### DISCUSSION TITLE
Modus [Silo laden] - Radlader wartet auf Abfahrer

### DISCUSSION THREAD

**Discussion by KlausFarmer** [Courseplay/Courseplay_FS22#3257] [Q&A]
Ich arbeite gerade daran, mein Durchfahrsilo (Bunker 2 auf der Eureka-Karte) mit dem größten Radlader mit der 8t Silagegabel (beides Vanilla) auf einen Anhänger zu entladen. Zunächst scheint die Arbeit über Courseplay 7.4.1.2 zu funktionieren, allerdings bleibt der Radlader spätestens nach der zweiten Schippe stehen und zeigt die Meldung "Warte auf Abfahrer".

Habe jetzt schon einige Zeit mit Courseplay und Autodrive 2.0.1.6 herumprobiert, aber keine Lösung gefunden. Hat jemand eine Idee, was ich da machen sollte?

**Comment by Tensuko** [comment]
Kannst du mir mal ein Bild von deinem Aufbau zeigen?
Soll das in einen Anhänger abgeladen werden oder in einen Trigger?

**Comment by KlausFarmer** [comment]
Hallo Tensuko,

 

vielen Dank für die schnelle Antwort. Unten ist ein Bild der ungefähren Situation. Aktuell ist das Silo leer, so dass ich es momentan nicht direkt nachstellen kann. Abladepunkt war der Anhänger im Bild.

 

Als ich heute den Traktor mit Anhänger auf die andere Seite der Straße hingestellt habe (ohne irgendwelche Autodrive-Haltepunkte oder -Kurse) wurde der Beladungsvorgang ohne Probleme durchgeführt.

 

Vielleicht ist mehr Abstand wichtig oder vielleicht stören die Autodrive Kurse und Punkte? Sorry, ich bin beim Ls22 recht unerfahren.

 

Im Moment hat es funktioniert, auch wenn ich den Anhänger manuell abfahren lassen und neu positionieren musste.

 

Habe ich etwas verkehrt gemacht oder benötigst Du mehr Input zu dem Problem? Kann halt ein paar Tage dauern, bis ich das Silo wieder gefüllt habe. Schlag mir einfach ein paar Experimente vor, die ich machen kann um das Problem zu klären.

 

Beste Grüße

Klaus

 

 



 

Von: Tensuko ***@***.***> 
Gesendet: Montag, 27. Mai 2024 07:32
An: Courseplay/Courseplay_FS22 ***@***.***>
Cc: KlausFarmer ***@***.***>; Author ***@***.***>
Betreff: Re: [Courseplay/Courseplay_FS22] Modus [Silo laden] - Radlader wartet auf Abfahrer (Discussion #3257)

 

Kannst du mir mal ein Bild von deinem Aufbau zeigen?
Soll das in einen Anhänger abgeladen werden oder in einen Trigger?

**Reply by Tensuko** [reply]
Ich kann leider keine Bilder sehen.... Bitte bei sowas nicht per Mail antworten, das geht in die Hose :)

**Comment by KlausFarmer** [comment]
Sorry,  ich hoffe jetzt kannst Du es sehen.

![Bild_2024-06-01_165321880](https://github.com/Courseplay/Courseplay_FS22/assets/170891156/ab06dfc0-ad83-43c7-9365-65009e368c3f)

**Reply by Tensuko** [reply]
Das sollte so passen. Eventuell den Abladepunkt anders setzen? Sollte ein blauer Kasten entstehen in dem Bereich sucht er den Trailer.

**Reply by KlausFarmer** [reply]
Es funktioniert jetzt. Zuvor habe ich den Abladepunkt auf den Trailer gesetzt. Nachdem ich den Abladepunkt auf das Zugfahrzeug setze funktioniert es.
